CONTRIBUTING.md

```diff
@@ -142,47 +144,95 @@ const mappers = {
 };
 ```
 
-### Jobs
+## 📊 Database Management
+
+### Using Seed Data
+
+You can populate the database with development data using:
 
-### Migrations
+```sh
+pnpm run --filter @argos/backend db:truncate && pnpm run --filter @argos/backend db:seed
+```
+
+### Migration
 
 #### Create a migration
 
 ```sh
 pnpm run --filter @argos/backend db:migrate:make my_migration
 ```
 
-#### Dump database
+#### Dump the database
 
 ```sh
 pnpm run --filter @argos/backend db:dump
 ```
 
-#### Execute the latest migration
+#### Apply the latest migration
 
 ```sh
 pnpm run --filter @argos/backend db:migrate:latest
 ```
 
-### Running the test suite
-
-You can reset the test database using:
+### Reset the Test database
 
 ```sh
 NODE_ENV=test pnpm run --filter @argos/backend db:reset
 ```
 
-## Coding style
+## ✅ Testing your changes
+
+### Linting
+
+Ensure your code follows the project’s coding standards:
+
+```sh
+pnpm run lint
+```
+
+### Unit and Integration Tests
+
+```sh
+pnpm run test
+```
+
+### End-to-End (E2E) Tests
+
+1. Install Playwright dependencies:
+
+```sh
+npx playwright install --with-deps
+```
+
+2. Run E2E setup:
+
+```sh
+pnpm run e2e:setup
+```
+
+3. Run E2E tests:
+
+```sh
+pnpm run e2e:start
+
+# or in debug mode with
+# pnpm run e2e:start -- --debug
+```
+
+## 📜 Coding Style
+
+Please follow the coding style of the current code base. Argos uses ESLint to maintain a consistent coding style. If possible, enable linting in your editor to get realtime feedback. Linting can be run manually with `pnpm run lint`.
+
+Continuous Integration will run linting on your PR, so it’s best to ensure your code is clean before submitting.
 
-Please follow the coding style of the current code base. argos uses eslint, so if possible, enable linting in your editor to get realtime feedback.
-Linting can be run manually with `pnpm run lint`.
+## 🚀 Roadmap and contribution ideas
 
-Finally, when you submit a pull request, linting is run again by Continuous Integration testing, but hopefully by then your code is already clean!
+Want to contribute but don’t know where to start? Check out [open issues](https://github.com/argos-ci/argos/issues) for ideas. Every contribution helps!
```

Comment:
Add a link to our roadmap here, also better to use discussions than issues. To rework.